src/Nethermind/Nethermind.Shutter/ShutterTxFilter.cs

```diff
@@ -0,0 +1,38 @@
+// SPDX-FileCopyrightText: 2024 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using Nethermind.Core;
+using Nethermind.Core.Specs;
+using Nethermind.Logging;
+using Nethermind.Consensus.Validators;
+using Nethermind.Consensus.Transactions;
+using Nethermind.TxPool;
+
+namespace Nethermind.Shutter;
+
+public class ShutterTxFilter(
```

Comment:
Not sure why this class exists, the interface was designed for tx pool, you can just have a method in `ShutterTxLoader`

Reply:
I did it this way because I wanted to have unit test for tx filtering - it seemed nicer for it to be its own class rather than testing an internal method